### DISCUSSION TITLE
Different channel A and B scan list mode

### DISCUSSION THREAD

**Discussion by f4axk** [egzumer/uv-k5-firmware-custom#299] [Ideas]
Channel A and B mode scan list
would it be possible to have a different choice of list
for A and B.
for example on A list1 and on B list2

Thank you for your work

**Comment by ErikS-web** [comment]
I don't think scanning is possible on both VFOs at the same time.
So where is the advantage?

You can easily switch between 1, 2 or * (both) during scanning

See [memory-channels-scanning](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Radio-operation#memory-channels-scanning)

**Reply by f4axk** [reply]
Thank you for the answer .
I didn't want to do a scan on both VFOs at the same time.
simply memorized list1 on A and list2 on B to change more
quickly from the list. On A I scan list 1 (RA relay), after I stop and go to B I scan list 2 (PMR).
Currently if we choose 1, 2 or * (both) it is for A and B


Thank you for your work on the firmware and drivers for chirp which works very well

**Reply by ErikS-web** [reply]
Yes ... scanning is run from Primary channel.
That is selected with Long-press `2 A/B` and marked on display by ►

**Comment by egzumer** [comment]
There are only 2 lists, independent from A or B
